packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-development.md

```diff
@@ -119,4 +109,26 @@ Example:
         <p>Content</p>
     </div>
 </div>
-```
\ No newline at end of file
+```
+
+## Screen reader considerations
+
+Just as front-end developers use their browser to see how their changes look, you should use a screen reader to see how your accessibility looks (we use Voice Over). You can use any screen reader that is available in your operating system. To test PatternFly, we target the following screen readers:
```

Comment:
The last line seems odd, especially because we don't mention any screen readers afterwards.

Reply:
Also, I'm wondering if we could update the one line about screen readers in your operating system? VO is found in the mac operating system, but common ones like NVDA and JAWS aren't, they're downloads for PC. Maybe we could say something like, "You can use a screen reader that is available in your operating system, like VO for Mac, or you could download an open source screen reader like [NVDA](https://www.nvaccess.org/download/) if you have a PC." Feel free to wordsmith that to sound better though. 😂